app/dashboard/utils.py

```diff
@@ -452,7 +452,7 @@ def record_user_action_on_interest(interest, event_name, last_heard_from_user_da
 
 def get_context(ref_object=None, github_username='', user=None, confirm_time_minutes_target=4,
                 confirm_time_slow=120, confirm_time_avg=15, confirm_time_fast=1, active='',
-                title='', update=None):
+                title='', update=None, gitcoin_tip_address=settings.GITCOIN_TIP_ADDRESS):
```

Comment:
Since we're not passing `gitcoin_tip_address` anywhere, can we simply use `setting.GITCOIN_TIP_ADDRESS` as the value for: `gitcoin_tip_address`

Reply:
Do I do that like you wanted?